mgmt/rest/rbody/v2/metric.go

```diff
@@ -0,0 +1,49 @@
+package v2
+
+import (
+	"fmt"
+
+	"github.com/intelsdi-x/snap/mgmt/rest/rbody"
+)
+
+type MetricList struct {
+	Metrics []rbody.Metric `json:"metrics,omitempty"`
+}
+
+type MetricReturned struct {
+	Metric *rbody.Metric `json:"metric,omitempty"`
+}
+
+func (m *MetricReturned) ResponseBodyMessage() string {
+	return "Metric returned"
+}
+
+func (m *MetricReturned) ResponseBodyType() string {
+	return "metric_returned"
+}
+
+type MetricsReturned MetricList
+
+func (m MetricsReturned) Len() int {
+	return len(m.Metrics)
+}
+
+func (m MetricsReturned) Less(i, j int) bool {
+	return (fmt.Sprintf("%s:%d", m.Metrics[i].Namespace, m.Metrics[i].Version)) < (fmt.Sprintf("%s:%d", m.Metrics[j].Namespace, m.Metrics[j].Version))
```

Comment:
I guess it was for sorting purpose, but I don't see any import of the package `sort`. It should be something like `sort.Sort(MyMetricsReturned)`.